lab_6_pipeline/pipeline.py

```diff
@@ -62,11 +115,23 @@ def __init__(
             corpus_manager (CorpusManager): CorpusManager instance
             analyzer (LibraryWrapper | None): Analyzer instance
         """
+        self._corpus = corpus_manager
+        self.analyzer = analyzer
 
     def run(self) -> None:
         """
         Perform basic preprocessing and write processed text to files.
         """
+        articles = self._corpus.get_articles().values()
+        doc_conllu = []
+        if self.analyzer:
+            doc_conllu = self.analyzer.analyze([article.text for article in articles])
+
+        for index, article in enumerate(articles):
```

Comment:
iterate over zipped object to simultaneously access article and its conllu content

Reply:
done. I found only one way to do this, in all the other cases, there was a Typeerror